templates/elasticsearch.yaml

```diff
@@ -25,18 +25,18 @@ spec:
         resources:
           # need more cpu upon initialization, therefore burstable class
           limits:
-            cpu: 1000m
+            cpu: 2000m
             memory: 2048Mi
           requests:
             cpu: 250m
             memory: 1536Mi
```

Comment:
See below. I'd prefer not bursting memory. i.e., set it to 2048Mi

Reply:
same as above: https://github.com/mesosphere/kubeaddons-configs/pull/16#discussion_r285500317

Reply:
OK. that's fair! Thanks for the pointer.